### PR TITLE
Initial version of the Core within Release

### PR DESCRIPTION
**Pull request by balhar-jakub**
Signed-off-by: Jakub Balhar <jakub@balhar.net>

### CODE REVIEW

File: Technical-Steering-Committee/release.md [hunk 1]
```diff
@@ -78,6 +78,15 @@ There are following parts build as a part of the release:
...
 - Source build
 
+### Core part of release
+
+Some parts of the Zowe are perceived as Core. The Core is a vendor neutral and outlined in this document. The packages are available from [zowe.org](https://www.zowe.org/download.html) unless stated otherwise. The packages are evolving internally and the evolution within them is for now considered part of the core. 
+
+- z/OS Convenience Build - Zowe x.x.x z/OS Convenience build 
+- z/OS SMP/E Build - Zowe 1.9.0 FMID AZWE001 and Zowe x.x.x PTF ptfId
+- CLI Core - Zowe x.x.x CLI Core
+- Zowe Explorer (Only downloadable from Visual Studio Code Marketplace)
```
Comment (Joe-Winchester) [inline Technical-Steering-Committee/release.md:88]: I like how the definition of Zowe core is tied back to the download site.  
For Zowe Explorer marketplace maybe add in the URL https://marketplace.visualstudio.com/items?itemName=Zowe.vscode-extension-for-zowe also a link, e.g. }
[Visual Studio Code Marketplace](https://marketplace.visualstudio.com/items?itemName=Zowe.vscode-extension-for-zowe).

**Review by 1000TurquoisePogs — APPROVED** [review]

**Review by janan07 — APPROVED** [review]
Edited and reviewed by Tech Info

**Comment by RASakach** [comment]
@balhar-jakub I have recommended changes:

**The “CORE” Attribute**
There are specific Zowe components that carry this designation. Core is loosely defined as a component of Zowe having the following characteristics:

- Is maintained by the Zowe community
- Is considered Vendor-neutral
- Is a package available for download from the Download page at https://Zowe.org unless otherwise directed via the  download section

This list identifies the specific Zowe V1 components designated as Core:

1. Zowe z/OS Convenience Build
    - Zowe v.r.mz* z/OS Convenience build 
    - Zowe z/OS SMP/E Build
2. Zowe 1.9.0 FMID AZWE001
    - Zowe v.r.m* PTF UOnnnnn** UOnnnnn**
3. Zowe CLI
    - Zowe v.r.m* CLI Core
4. Zowe Explorer
    - Listed as Visual Studio Code Marketplace (All versions available at https://marketplace.visualstudio.com/items?itemName=Zowe.vscode-extension-for-zowe ) 

*Where  v.r.m represents the continuous Zowe Version, Release, Maintenance levels associated with the specific packages.
** Where nnnnn represents a specific number associated with the PTF as listed on the download page.

**Review by zFernand0 — APPROVED** [review]
LGTM!
Vote: Yes 😋

**Pull request merged**